eva-accession-release/src/main/java/uk/ac/ebi/eva/accession/release/runner/AccessionReleaseJobLauncherCommandLineRunner.java

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
Since this program is the same as the commandlinerunner from other modules, could we share code across the classes?

Reply:
the thing is that several pipelines might need different CLI features. these two are the same, but use different parameters, so we could make the class generic on the InputParameters, and if we need to add something like a CLI parameter (not on the properties file), we will have to extend the class and make constructors, overload one method or two, and at that point it will be exactly as it is now.

What I mean is, it's good to extract the same behaviours to a common place if you want to keep the behaviours parallel, but I don't think that's the scenario here.